UIPickerExtensions.swift

```diff
@@ -0,0 +1,57 @@
+import UIKit
+import ViewModelsSample
+
+extension UIPickerView {
+    private struct AssociatedKeys {
+        static var elementsKey = UnsafeMutablePointer<Int8>.allocate(capacity: 1)
+    }
+
+    private var elements: [String]? {
+        get {
+             return objc_getAssociatedObject(self, AssociatedKeys.elementsKey) as? [String]
+        }
+        set {
+            objc_setAssociatedObject(self, AssociatedKeys.elementsKey, newValue, objc_AssociationPolicy.OBJC_ASSOCIATION_RETAIN)
+        }
+    }
+    
+    public var pickerViewModel: PickerViewModel? {
+        get { return trikotViewModel() }
+        set(value) {
+            guard let pickerViewModel = value else { return }
```

Comment:
You should also bind the View's viewModel to inherit VIewModel's base properties like`hidden` and `backgroundColor` for example.

`viewModel  = pickerViewModel`